### ISSUE TITLE
Modal panels should get focus when opened

### ISSUE THREAD

**Issue by respinos** [mlibrary/cozy-sun-bear#84]
Currently the get no focus, which is lame.

**Comment by respinos** [comment]
Fixed by PR #86
